src/functional/resources/sample.csv

```diff
@@ -1,9 +1,9 @@
-Organization,Repository,branch,Author's GitHub ID,Author's Display Name,Author's Git Author Name
-reposense,testrepo-Beta,master,nbriannl,Nbr,
-reposense,testrepo-Beta,master,zacharytang,Zac,Zachary Tang
-reposense,testrepo-Beta,master,April0616,Fan,LAPTOP-7KFM2KSP\User;Fan Yuting
-reposense,testrepo-Beta,master,CindyTsai1,Cin,YuHsuan
-reposense,testrepo-Charlie,master,charlesgoh,Cha,Charles Goh
-reposense,testrepo-Charlie,master,Esilocke,Esi,
-reposense,testrepo-Charlie,master,jeffreygohkw,Jef,Jeffrey Goh
-reposense,testrepo-Charlie,master,wangyiming1019,Wan,ACER\kyle;Wang Yiming
+Organization,Repository,branch,Author's GitHub ID,Author's Display Name,Author's Git Author Name,IgnoreGlobList
+reposense,testrepo-Beta,master,nbriannl,Nbr,,
+reposense,testrepo-Beta,master,zacharytang,Zac,Zachary Tang,
+reposense,testrepo-Beta,master,April0616,Fan,LAPTOP-7KFM2KSP\User;Fan Yuting,
+reposense,testrepo-Beta,master,CindyTsai1,Cin,YuHsuan,
```

Comment:
Quick question: how do you plan to handle the setting where different set of IgnoreGlobLists entered for each individual author from same repository?

Currently it seems that the latest author will always override the existing `IgnoreGlobList`

Reply:
Hmmm, as we currently are unable to set a 'repo level' setting in the current `.csv` format, we probably just need the user to ensure the `IgnoreGlobList` is set the same for all users, or for the last user.
This should be resolved when we shift over to `config.json` in #232, so maybe we can instead postpone adding this to the `config.csv` for now?

Reply:
1. We need to add support for repo-config.csv soon. 
2. How about this? If something is specified both at repo level and author level, author level _modifies_ (not _replaces_) repo level (cumulative).

Reply:
Since we don't have the repo-config.csv yet, for this PR we will change it to do the analysis and file exclusions by author level?

Reply:
I guess you can do that. Alternatively, as the csv file is not a backend matter, you can _assume_ there is a way to specify file exclusions at repo level and provide backend support for it. That way, @yong24s can add the support for repo-config.csv without needing any further changes from you. Otherwise the two of you will have to work together to implement support for file exclusion at repo level.

Reply:
The thing is if we are to support file exclusion at author level, we would need to modify the `FileInfo` extraction/analysis and `CommitInfo` extraction to be done at the author level as well (which is currently at repo level).
I think that for this PR, we will just assume and stick at repo level exclusion, and modify the `FileInfo` extraction and `CommitInfo` extraction to be at author level in another PR?